manual/docs/machinery-compare.1.md

```diff
@@ -3,7 +3,7 @@
 
 ## SYNOPSIS
 
-`machinery compare` [-s SCOPE | --scope=SCOPE] [-e EXCLUDE-SCOPE | --exclude-scope=EXCLUDE-SCOPE] [--no-pager] [--show-all] [--html] NAME1 NAME2
+`machinery compare` [-s SCOPE | --scope=SCOPE] [-e IGNORE-SCOPE | --ignore-scope=IGNORE-SCOPE] [--no-pager] [--show-all] [--html] NAME1 NAME2
```

Comment:
Maybe that existing `-i` can be replaced by `-s` since it is most likely less commonly used and "s" is also the first later of the full name.